### PR TITLE
Use the exisiting banner paragraph instead of a new field

### PR DESCRIPTION
**Pull request by msayoung**

### CODE REVIEW

**Comment by msayoung** [comment]
Once this is merged we'll also need : https://github.com/localgovdrupal/localgov_microsites/pull/172

File: config/install/core.entity_form_display.node.localgov_page.default.yml [hunk 1]
```diff
@@ -8,19 +8,25 @@ dependencies:
     - node.type.localgov_page
   module:
     - layout_paragraphs
-    - media_library
-    - text
+    - paragraphs
+_core:
+  default_config_hash: 5n68a59XgYB8X6k6JgpGhPsD8RY7EHG8bdpLZKZFEes
```
Comment (finnlewis) [inline config/install/core.entity_form_display.node.localgov_page.default.yml]: @msayoung I think we don't want this default_config_hash line in.

File: config/install/core.entity_form_display.node.localgov_page.default.yml [hunk 2]
```diff
@@ -32,7 +38,7 @@ content:
       require_layouts: 1
     third_party_settings: {  }
   localgov_page_summary:
-    type: text_textarea
+    type: string_textarea
```
Comment (finnlewis) [inline config/install/core.entity_form_display.node.localgov_page.default.yml:39]: @ekes and @msayoung - I've just seen the same thing here as in https://github.com/localgovdrupal/localgov_page/pull/19#pullrequestreview-1026386842

Is this a default widget config change in the latest version of Drupal perhaps?

Comment (finnlewis) [reply config/install/core.entity_form_display.node.localgov_page.default.yml:39]: We think this is correct, as the base field is string_long. See https://github.com/localgovdrupal/localgov_page/blob/1.x/config/install/field.storage.node.localgov_page_summary.yml#L9

**Comment by msayoung** [comment]
@finnlewis review please ? :)

**Review by finnlewis — APPROVED** [review]
Tested locally, works a treat, thanks @msayoung !

**Pull request merged**